concepts/index.md

```diff
@@ -195,3 +195,16 @@ specify which events should trigger sending a message to your URL.
 
 This feature is still in beta so please send a request to support@shipcloud.io
 for it to be made available for you.
+
+## Testing
+There are a lot of tools out there for testing webhooks. If you'd like a starting point check out
+this blogpost called [_60+ Tools and Services for API and Webhook Logging, Debugging, Testing,
+Monitoring, Documentation and Discovery_](http://john-sheehan.com/blog/ultimate-api-webhook-backend-service-debugging-testing-monitoring-and-discovery-tools-list).
+We'd recommend using a service like [requestb.in](http://requestb.in) or [mocky.io](http://mocky.io)
```

Comment:
My goal was to show free tools. requestb.in is a free tool by runscope so if you want to do more than "just mock" the API you'd go over there anyways.